packages/node_modules/@ciscospark/spark-core/src/lib/batcher.js

```diff
@@ -167,11 +168,16 @@ const Batcher = SparkPlugin.extend({
    * @returns {Promise<undefined>}
    */
   handleHttpError(reason) {
-    const msg = reason.message || reason.body || reason;
-    return Promise.all(reason.options.body.map((item) => this.getDeferredForRequest(item)
-      .then((defer) => {
-        defer.reject(msg);
-      })));
+    if (reason instanceof SparkHttpError) {
+      if (has(reason, `options.body.map`)) {
+        return Promise.all(reason.options.body.map((item) => this.getDeferredForRequest(item)
+          .then((defer) => {
+            defer.reject(reason);
```

Comment:
@ianwremmel ready for one more round. I changed this to reject with the http error as we discuss as well.